keyboards/xelus/snap96/keymaps/default/keymap.c

```diff
@@ -0,0 +1,24 @@
+// Default layout for Snap96
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
+
```

Comment:
QMK defines this automatically.

https://github.com/qmk/qmk_firmware/blob/e34764502f7fdd0e1104b18fb4ddf77a352ee2a7/quantum/quantum_keycodes.h#L29-L31

```suggestion
```